lua/modules/hyperspace.i

```diff
@@ -2173,6 +2173,8 @@ playerVariableType playerVariables;
 %rename("%s") ResourceControl::RenderImage;
 %rename("%s") ResourceControl::RenderImageString;
 
+%rename("%s") ResourceControl::LoadFile;
```

Comment:
Do we want this exposed? This opens a resource file and returns its contents (I think) as a char* (or char array).

Also, LoadFile allocates new memory (for the returned string); how are we managing this? I don't want to expose a method if it is going to leak memory every time it is called.

Reply:
Ran a quick test to check if `%newobject ResourceControl::LoadFile;` plugs the leak and it does. Loading blueprints every frame without it causes the game to crash in a few seconds. Loading blueprints every frame _with_ it does not crash the game, and checking the memory usage in task manager confirms that it doesn't increase.

Reply:
Okay. Let's hope the delete[] on the LoadFile result works properly on both Windows and Linux.